Openshift-EE/pipelines/ElasticSearch/stages/3-functional/86TL-App-replica-scaleup/app-replica-scaleup

```diff
@@ -59,13 +59,19 @@ sed -i '/command:/i \
 echo "Running the litmus test for Elasticsearch Deployment Scaleup.."
 cat run_scaleup_test.yml
 
-bash ../Openshift-EE/utils/litmus_job_runner label='app: elasticsearch-deployment-litmus' job=run_scaleup_test.yml
+bash ../Openshift-EE/utils/litmus_job_runner label='app:elasticsearch-deployment-litmus' job=run_scaleup_test.yml
```

Comment:
Label needs to be modified. This has been used for deployer already

Reply:
Ok